src/gurt/dlog.c

```diff
@@ -636,7 +637,10 @@ void d_vlog(int flags, const char *fmt, va_list ap)
 	 * NB: flush to logfile if the message is important (warning/error...)
 	 * or the last flush was 1+ second ago.
 	 */
-	flush = (lvl >= DLOG_WARN) || (tv.tv_sec > last_flush);
+	if (mst.flush_pri == DLOG_DBG)
+		flush = true;
+	else
+		flush = (lvl >= mst.flush_pri) || (tv.tv_sec > last_flush);
 	if (flush)
```

Comment:
Since mst.flush_pri is setup to DLOG_DBG by default, why go you need the first test.?

Reply:
its not set to debug by default, it is set to warn
'lvl' here in case of debug can be multi-bit setting; when debug flush priority is selected we always flush as debug is our lowest setting

Reply:
So why do you need the first test.   Wouldn't it always be true for the test "lvl >= mst.flush_pri" when flush_pri is set to DLOG_DBG?